src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -84,6 +91,8 @@ public function setInputStream($stream)
     /**
      * Returns the helper's input stream.
      *
+     * @deprecated Will be removed in 4.0, use StreamableInputInterface::getStream() instead
```

Comment:
I was more talking about renaming the method in the interface to getInputStream and setInputStream, would that work?